modules/installation-configuration-parameters.adoc

```diff
@@ -1046,7 +1046,7 @@ Additional {rh-virtualization} configuration parameters for machine pools are de
 |Integer
 
 |`<machine-pool>.platform.ovirt.instanceTypeID`
-|Optional. An instance type UUID, such as `00000009-0009-0009-0009-0000000000f1`, which you can get from the `https://<engine-fqdn>/ovirt-engine/api/instancetypes` endpoint.
+|Optional. An instance type UUID, such as `00000009-0009-0009-0009-0000000000f1`, which you can get from the `https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api/instancetypes` endpoint.
```

Comment:
This one looks a bit odd in the preview with a capital letter in the variable:
` the https://<Manager-fqdn>/ovirt-engine/api/instancetypes endpoint.`

Reply:
You're right. I'm changing all those instances back to "engine-fqdn".